lambdas/service/app.py

```diff
@@ -321,6 +321,13 @@ def get_summary():
     """
     # Setup logging
     logger = logging.getLogger("dashboardService.webservice.get_summary")
+    # Determine index
```

Comment:
`assert entity_type in ('specimens', 'files')`

Reply:
And assert is wrong. Assert generates a 500 status code being sent to the client. It should be a 400 status code.

Reply:
Implemented @GPelayo solution to this problem.